src/inventory.gql

```diff
@@ -136,5 +163,14 @@ input UpdateProductItemInput {
   productVariantId: UUID!
 
   """The inventory state of the product item"""
-  isInInventory: Boolean!
+  inventoryStatus: ProductItemStatus!
+}
+
+"""The input to reserve a batch of product items"""
+input ReserveProductItemsBatchInput {
+  """The product variant id of the product item"""
+  productVariantId: UUID!
+
+  """The number of product items to reserve"""
+  number: Float!
```

Comment:
maybe this can be changed to Integer

Reply:
Done. I hope, i dont need floats in the future :smile: